Author: Joerg Dorchain <joerg@dorchain.net>
Description: Add tcpkill support for handle ppp interfaces.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/pcaputil.c
+++ b/pcaputil.c
@@ -52,6 +52,9 @@
 	case DLT_NULL:
 		offset = 4;
 		break;
+	case DLT_LINUX_SLL:   /* e.g. ppp */
+		offset = 16;
+		break;
 	default:
 		warnx("unsupported datalink type");
 		break;
